test/jsonValidationTest.js

```diff
@@ -45,11 +45,11 @@ describe('Validating Cohorts...', () => {
 });
 
 describe('Validating Workbooks...', () => {
```

Comment:
while you are touching this do you want to fix #89 ?  it's seems to be a 1 line fix in here.

Reply:
I will handle it separately. Not sure there is an easy way to detect BOM.

Reply:
as I commented in that thing, it's a one line change:

> from an ancient archive of nodejs 0.x:
> nodejs/node-v0.x-archive#1918
> it looks like the fix is to just strip it out after reading from the file:
> `body = body.replace(/^\uFEFF/, '');`

Reply:
To unblock Scott's team I will deal with it separately since it may require more testing and investigation as we discussed.